README.txt

```diff
@@ -1 +1,24 @@
-## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
\ No newline at end of file
+## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
+
+**How to Earn XP**
+
+**Earn Activity XP**
```

Comment:
I think we also want to leave the door open for arbitrary XP grants based on nominations? e.g. I organized this event at NFT NYC and think I should get Y